.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2546,7 +2547,7 @@ stages:
         command: "CheckBuildLogsForErrors"
 
 - stage: nuget_installer_smoke_tests
-  condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'))
+  condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'), eq(variables['isRunAllInstallerTestsBuild'], 'True'))
```

Comment:
`variables['isBenchmarksOnlyBuild'], 'False'` is useless now no? as benchmarks never run on master and `isRunAllInstallerTestsBuild` only on master (or overriden)

Reply:
> variables['isBenchmarksOnlyBuild'], 'False' is useless now no? as benchmarks never run on master and isRunAllInstallerTestsBuild only on master (or overriden)

benchmarks _sometimes_ run on master (the nightly run) and `isRunAllInstallerTestsBuild` doesn't _necessarily_ run on master 😉 so I think these two are complementary